Crashlytics/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # v7.4.0
 - [changed] Removed obsolete crash reporting mechanism from the SDK (#7076).
+- [changed] Moved some binary image work to be done asynchronously to improve start-up performance (#7332).
```

Comment:
Think this needs to be under Unreleased section

Reply:
yep my bad